tasks.py

```diff
@@ -19,14 +26,31 @@ def remove_check_file(path):
     if os.path.isfile(path):
         os.remove(path)
 
+def run_as(os_name):
+    if os_name.startswith('linux') or os_name == 'darwin':
```

Comment:
1. Although highly unlikely for someone to install yaksh on FreeBSD, it does allow user to install sudo and run stuff as sudo-user(Users on bsd have to install sudo). 
2. Also, ```sys.platform``` on FreeBSD returns freebsd(x) (x being the version of bsd, eg freebsd8). We can add a freebsd support for this by just adding another condition for it

Reply:
Adding an Or condition with ```or os_name.startswith('freebsd'):```

I am adding the output ```freebsd``` based on your recommendation (as I do not have a FreeBSD system to test this)